value_test.go

```diff
@@ -663,7 +663,7 @@ func TestReadOnlyOpenWithPartialAppendToWAL(t *testing.T) {
 		{Key: k2, Value: v2},
 	})
 	buf = buf[:offset-6]
-	require.NoError(t, ioutil.WriteFile(vlogFilePath(dir, 1), buf, 0777))
```

Comment:
This test is returning error for memtable even when we have not written any data. Please fix this @NamanJain8

Reply:
Fixed this.